packages/web/config/ibc-assets.ts

```diff
@@ -963,6 +963,12 @@ export const IBCAssetInfos: (IBCAsset & {
     sourceChannelId: "channel-326",
     destChannelId: "channel-5",
     coinMinimalDenom: "stustars",
+  },
+   {
```

Comment:
please move to the bottom of the list

Reply:
Thank you, just added the chain to chain-infos.ts and added the token logos.